languages/js/src/Messages.ts

```diff
@@ -0,0 +1,12 @@
+export interface Message {
+  kind: String;
+  msg: String;
+}
+
+export function processMessage(message: Message) {
+  if (message.kind === 'Print') {
+    console.log(message.msg);
+  } else if (message.kind === 'Warning') {
+    console.warn(message.msg);
+  }
```

Comment:
I think this will be more extensible as a `switch` statement:
```suggestion
  switch (message.kind) {
    case 'Print':
      console.log(message.msg);
      break;
    case 'Warning':
      console.warn(message.msg);
      break;
  }
```

Reply:
👍